src/controller/request/debtor-request.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ *  SudoSOS back-end API service.
+ *  Copyright (C) 2020  Study association GEWIS
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @typedef HandoutFinesRequest
+ * @property {Array<integer>} userIds.required - Users to fine. If a user is not eligible for a fine, a fine of 0,00 will be handed out.
+ * @property {string} referenceDate - Reference date on to calculate balance for (and "now")
```

Comment:
This sentence is not correct I believe. Also is the default "now"?

Reply:
Fixed in https://github.com/GEWIS/sudosos-backend/pull/42/commits/1261bbd43f4ec807b1443e718464cf93e1115b4a. Balances for fines are calculated on the reference date and now. If a user is in debt at both points in time, they are eligible for a fine.